src/config/settings.js

```diff
@@ -91,6 +91,19 @@ function init() {
   return settings;
 }
 
+async function isFirstRun() {
+  let settingsPath = join(paths.config(), 'settings.json');
+  try {
+    await util.promisify(fs.access, fs.constants.F_OK)(settingsPath, fs.constants.F_OK);
```

Comment:
Nothing in the docs seems to indicate that `util.promisify` can take more than one argument?

(We could also use the [pre-promisified version](https://nodejs.org/api/fs.html#fs_fspromises_access_path_mode) if you want.)